linode_api4/objects/vpc.py

```diff
@@ -19,31 +33,11 @@ class VPCSubnet(DerivedBase):
         "id": Property(identifier=True),
         "label": Property(mutable=True),
         "ipv4": Property(),
-        "linodes": Property(),
+        "linodes": Property(json_object=VPCSubnetLinode),
```

Comment:
Side note: I wonder if there's a way we could explicitly specify that this should be a list value.